systemtests/src/test/java/io/enmasse/systemtest/sharedinfra/MessagingInfraTest.java

```diff
@@ -173,8 +189,13 @@ public void testInfraRestart() throws Exception {
         LOGGER.info("Waiting for pods to come back up");
 
         // Give operator some time to detect restart and re-sync its state
-        Thread.sleep(60_000);
+        Thread.sleep(120_000);
```

Comment:
is it posible to do it some more deterministic way? Like monitor logs of operator and react on reconcile?

Reply:
@kornys I'm not sure... I think maybe if we recorded some reconnect count for each router and broker and waited for that to increase. That would be the best way to know deterministically that the operator know that they were restart. But I'm not sure how useful it is in practice given that one already has restart counts as part of Kubernetes itself.

Reply:
ok leave it as is :)